crates/iota-types/src/unit_tests/multisig_tests.rs

```diff
@@ -434,7 +434,7 @@ fn test_zklogin_in_multisig_works_with_both_addresses() {
         .into_iter()
         .collect();
 
-    let aux_verify_data = VerifyParams::new(parsed, ZkLoginEnv::Test, true, Some(30));
+    let aux_verify_data = VerifyParams::new(parsed, vec![], ZkLoginEnv::Test, true, true, Some(30));
```

Comment:
Second parameter doesn't exist on our side, it's `supported_providers`. You can remove that.